feedly/api_client/data.py

```diff
@@ -76,6 +76,17 @@ def tag_entries(self, entry_ids: List[str]):
         self._client.do_api_request(f'/v3/tags/{quote_plus(self["id"])}', method='put',
                                     data={'entryIds': [entry_id for entry_id in entry_ids]})
 
+    def untag_entry(self, entry_id: str):
+        self.untag_entries([entry_id])
+
+    def untag_entries(self, entry_ids: List[str]):
+        # limitation due to the url length: articles are "de-tagged" by batch of 50.
+        for i in range(0, len(entry_ids), 50):
+            self._client.do_api_request(
+                f'/v3/tags/{quote_plus(self["id"])}/{",".join([quote_plus(d) for d in entry_ids[i: i+50]])}',
```

Comment:
i think you can do 
POST /v3/entries/.mdelete 
with the content being the list of entry ids.

Reply:
This is not on the API ? https://developer.feedly.com/v3/tags/#delete-tags
How do you precise which tag to delete then ?

Reply:
you're right, .mdelete is not available on this api endpoint